lib/ecto/cron.ex

```diff
@@ -1,6 +1,6 @@
 # This is copypasta from https://github.com/jshmrtn/crontab/blob/master/lib/crontab/cron_expression/ecto_type.ex
 
-# TODO: Replace with simple Code.ensure_compiled as soon as Elixir mininum
+# TODO: Replace with simple Code.ensure_compiled as soon as Elixir minimum
```

Comment:
The place I copypasta-ed it from (appreciate that copypasta is in your dictionary!) fixed the typo 5 months ago: https://github.com/jshmrtn/crontab/pull/87/files#diff-0c818cfd73c951537bbca52eb46667d2a81a75d0d8d171f45c3f1abf156114e0

Reply:
It isn't technically in the dictionary, it's in the expect list which is generated in the process of finding words that aren't properly spelled: https://github.com/jsoref/bikebrigade-dispatch/blame/spell-check/.github/actions/spelling/expect.txt#L55

Essentially any word not in the dictionary is offered to Google Sheets, any time it offers a suggested replacement, I think about it for a split-second.
* If I like the replacement, it'll be replaced.
* If not, I think about whether I have an alternate replacement.

Anything that doesn't have a suggestion can just sit there unchanged. I may (or may not) review the remainder for additional items to fix (I think in this case I did).

If someone wants to keep using the workflow after the spelling PR, they can either keep the items as `expect`ed in which case if they disappear, they'll eventually be removed, or add them to the dictionary by moving them to an `allow`ed file (which is merged into the dictionary).